### PR TITLE
mocks: Name all mock runtimes as 'Runtime'

### PR DESCRIPTION
**Pull request by NunoAlexandre**
Currently, some of the mocks' runtimes are named 'Test' and others 'MockRuntime'. Given that that struct is effectively a runtime AND it is defined in a `mock.rs` file, naming it `Runtime` is sufficient, making it consistent across all mocks and avoiding naming redundancy.

### CODE REVIEW

**Review by mustermeiszer — DISMISSED** [review]
That's a nice cleanup. Makes copy-paste-coding the test easier ^^

File: pallets/interest-accrual/src/tests.rs [hunk 1]
```diff
@@ -17,7 +17,7 @@ use sp_runtime::{
 };
 
 use crate::{
-	mock::{Rate, Test as TestConfig},
+	mock::{Rate, Runtime as RuntimeConfig},
```
[the remark this answers is not in the document]
Comment (NunoAlexandre) [reply pallets/interest-accrual/src/tests.rs]: Good catch. There wasn't a collision actually, not sure why we had the alias. It's just `Runtime` now 👍

**Review by mustermeiszer — APPROVED** [review]
Let's get this in. Thanks!

**Comment by NunoAlexandre** [comment]
Thanks @mustermeiszer 😊

**Pull request merged**